### PR TITLE
New borg sounds

### PR DESCRIPTION
**Pull request by MilenVolf**
## About the PR
No more PAI borgs.

Adds new voice for borgs, plus manic laugher for assault borg to play with the NT prey.

## Why / Balance
The feature we never wanted, but needed.

## Media

# Standard borg
https://github.com/space-wizards/space-station-14/assets/63782763/534b0898-0bba-4002-8d14-0777396b21d4

# Syndieborg
https://github.com/space-wizards/space-station-14/assets/63782763/fe236718-98f1-4a37-b219-925f98427c46

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl:
- add: Borgs now have brand new voice and walking sounds.
- add: Syndicate assault borgs now have new feature to scare - manic laugher.

### CODE REVIEW

File: Resources/Audio/Voice/Talk/Silicon/attributions.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+- files: ["borg.ogg", "borg_ask.ogg", "borg_exclaim.ogg"]
+  license: "CC-BY-SA-4.0"
+  copyright: "Recorded and mixed by https://github.com/MilenVolf"
+  source: "https://git.arumoon.ru/Workbench-Team/space-station-14/-/merge_requests/123"
```
Comment (metalgearsloth) [inline Resources/Audio/Voice/Talk/Silicon/attributions.yml:4]: This isn't even public and I don't think that's even their license.

Comment (MilenVolf) [reply Resources/Audio/Voice/Talk/Silicon/attributions.yml:4]: Oh, yeah

Comment (MilenVolf) [reply Resources/Audio/Voice/Talk/Silicon/attributions.yml]: It was made by me so... What license should I put in?

**Comment by slarticodefast** [comment]
I always liked the cute and innocent sounds the borgs made. Maybe only use it for the syndie borg? It is perfect for them.

**Comment by Hanzdegloker** [comment]
I like the current voice, it fits borgs and PAIs very well IMO

**Comment by Froozigiusz** [comment]
I guess those sounds can be used for a syndicate cyborg or the main station artificial intelligence (once it's added), to further differenciate them from normal sillicons. TaxiBots (and SimpleBots in general), personal Artifitial Intelligence, and normal cyborgs ar meant to be „small” relatively, and those sounds are quite threatening to listen to.

**Comment by MilenVolf** [comment]
> I guess those sounds can be used for a syndicate cyborg or the main station artificial intelligence (once it's added), to further differenciate them from normal sillicons. TaxiBots (and SimpleBots in general), personal Artifitial Intelligence, and normal cyborgs ar meant to be „small” relatively, and those sounds are quite threatening to listen to.
> 
> Also walking sounds are just not it. I do not like them and I assume others would also agree.

If we are talking about station AI speech through hologram, then it should sound not robotic, but in a light-waved synth way I think

**Comment by MilenVolf** [comment]
> I guess those sounds can be used for a syndicate cyborg or the main station artificial intelligence (once it's added), to further differenciate them from normal sillicons. TaxiBots (and SimpleBots in general), personal Artifitial Intelligence, and normal cyborgs ar meant to be „small” relatively, and those sounds are quite threatening to listen to.

But yeah, I think we just used to cute little innocent pai voice that borgs use.

**Comment by MilenVolf** [comment]
> Also walking sounds are just not it. I do not like them and I assume others would also agree.

I am open to any suggestions, so if don't like 'em, then suggest any other sound or maybe just the picture of you seeing it, so maybe'll try to capture it in a sound.

But the sound of working servos is the first thing that came into my mind, so I did my best to make it sound good.

**Comment by Golinth** [comment]
The walking sound is too… sharp? It will get very grating as a Borg after walking around for a while. I honestly don’t have any good suggestions, other than I just don’t think it will work well as is. 

I’m also in agreeance with the others. The syndie Borg sounds perfect, literally 10/10 imo perfectly menacing, but the normal Borg should stay as is. It sounds a bit too deep and evil to have a Borg sound like that by default. Maybe pitch it up to closer match the current sound?

**Comment by MilenVolf** [comment]
> The walking sound is too… sharp? It will get very grating as a Borg after walking around for a while. I honestly don’t have any good suggestions, other than I just don’t think it will work well as is.
> 
> I’m also in agreeance with the others. The syndie Borg sounds perfect, literally 10/10 imo perfectly menacing, and if possible only the normal Borg should change. It sounds a bit too deep and evil to have a Borg sound like that by default. Maybe pitch it up to closer match the current sound?

Okay, will try higher pitched version.

**Comment by MilenVolf** [comment]
I've changed sounds a little bit. Walking sound is now less louder, so should not be so frustrating for borg players after a while.

# Borg
https://github.com/space-wizards/space-station-14/assets/63782763/85649683-ab14-4ad1-9d46-0e56ad5c04f0

# Syndieborg
https://github.com/space-wizards/space-station-14/assets/63782763/c10d1025-9baf-4b07-8baa-41b8a898eff4

I made syndieborg voice a little deeper to better match his laugher.

**Comment by UbaserB** [comment]
The normal borg voice is too jarring, should be softer imo

**Comment by Djungelskog2** [comment]
the base borg talking sound just sounds really irritating, Id stick with a softer version of the first version

**Comment by mirrorcult** [comment]
i quite enjoy the borg talking sound

**Comment by mirrorcult** [comment]
needs to be quieter i think though but the sounds are good

**Comment by MilenVolf** [comment]
> needs to be quieter i think though but the sounds are good

Okay, will do

**Comment by MilenVolf** [comment]
Another portion of borgs.
All sounds are now quieter. Found a way to make base borg voice softer and less high-pitched noisy.
# Borg
https://github.com/space-wizards/space-station-14/assets/63782763/e5ebc619-c375-43b3-87ef-e85ce0d99c88

# Syndieborg
https://github.com/space-wizards/space-station-14/assets/63782763/0658fad2-e46e-4884-a30e-134eec490bf9

**Comment by Froozigiusz** [comment]
After hearing the newest version, I suppose it is okay to be in–game. The  initial boom in laughter of syndicate cyborg is still a bit jarring though.

**Comment by Golinth** [comment]
New version is pretty much perfect IMO.

**Comment by MilenVolf** [comment]
> This is really good. One thing i would like though is for the walking sound to be constant, since it seems like the borgs travel with treads or wheels of some sort, so if it was a full whirr as they moved it would be a little less annoying.

Okay, will try this out

**Comment by MilenVolf** [comment]
Wheel walking sound:

https://github.com/space-wizards/space-station-14/assets/63782763/8cbaeb3b-e928-41a6-8db5-702ce27851cb

Bonus: medborg hover-walking sound (just a chaotic thought came into my mind):

https://github.com/space-wizards/space-station-14/assets/63782763/7940d424-caa5-4596-9ae1-6200074b8701

File: Resources/Audio/Effects/Footsteps/attributions.yml [hunk 2]
```diff
@@ -75,7 +75,6 @@
 - files:
   - borgwalk1.ogg
   - borgwalk2.ogg
-  - borgwalk3.ogg
   license: "CC-BY-SA-4.0"
-  copyright: "Taken from plingativator freesound.org and mixed by https://github.com/MilenVolf"
-  source: "https://freesound.org/people/plingativator/sounds/188917/"
+  copyright: "Taken from IENBA freesound.org and modified by https://github.com/MilenVolf"
+  source: "https://freesound.org/people/IENBA/sounds/697379/"
```
Comment (MilenVolf) [inline Resources/Audio/Effects/Footsteps/attributions.yml:80]: Yes, you got it right, both samples from a single audio. First one was just heavily processed.

**Review by mirrorcult — APPROVED** [review]

**Comment by DummeHytteOst** [comment]
> This is really good. One thing i would like though is for the walking sound to be constant, since it seems like the borgs travel with treads or wheels of some sort, so if it was a full whirr as they moved it would be a little less annoying.

it really depends on the borg, i think the walking sound would fit the engi borg a lot.

the Engi borg seems to be walking, and i believe its inspired by the droideka from star wars, just without all the combat and rolly poley jazz.

the service borg have legs (the one that looks like a butler), but i don't think the walking sound that was presented in this pr would be well suited for it. 
i don't remember if terminators have a unique footstep sound for their metal form, but it might be better for service borgs? 

the jani borg does appear to have wheels, don't really see this one needing a new one after this pr ~~maybe sqeaky wheels once in a while~~

The salvager borg does have threads, and i feel a more bulky and heavy sound could be used for it, since its not like the agile lightweight science borg, but a heavy and bulky hauler/industrial build.

i do hope that these borg types gets a more appropriate sounds, as all of them have unique features, and widely different builds.  

@MilenVolf

**Comment by MilenVolf** [comment]
> > This is really good. One thing i would like though is for the walking sound to be constant, since it seems like the borgs travel with treads or wheels of some sort, so if it was a full whirr as they moved it would be a little less annoying.
> 
> it really depends on the borg, i think the walking sound would fit the engi borg a lot.
> 
> the Engi borg seems to be walking, and i believe its inspired by the droideka from star wars, just without all the combat and rolly poley jazz.
> 
> the service borg have legs (the one that looks like a butler), but i don't think the walking sound that was presented in this pr would be well suited for it. i don't remember if terminators have a unique footstep sound for their metal form, but it might be better for service borgs?
> 
> the jani borg does appear to have wheels, don't really see this one needing a new one after this pr ~maybe sqeaky wheels once in a while~
> 
> The salvager borg does have threads, and i feel a more bulky and heavy sound could be used for it, since its not like the agile lightweight science borg, but a heavy and bulky hauler/industrial build.
> 
> i do hope that these borg types gets a more appropriate walking sounds, as all of them have unique features, and widely different builds.

Thank you for the comment and paying attention to such details.
I'll try to come up with something for the walking sounds for other Borgs.

**Pull request merged**